geode-core/src/main/java/org/apache/geode/cache/configuration/CacheLoaderType.java

```diff
@@ -57,15 +55,16 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "cache-loader-type", namespace = "http://geode.apache.org/schema/cache",
-    propOrder = {"className", "parameter"})
+    propOrder = {"className", "parameters"})
```

Comment:
It would be nice that for every POJO change we made, we would have a junit test to make sure the resulting xml still pass the xsd validation.